spring-cloud-stream-binder-kafka-docs/src/main/asciidoc/overview.adoc

```diff
@@ -181,4 +186,35 @@ Exercise caution when configuring both `minPartitionCount` for a binder and `par
 If a topic already exists with a smaller partition count and `autoAddPartitions` is disabled (the default), then the binder will fail to start.
 If a topic already exists with a smaller partition count and `autoAddPartitions` is enabled, new partitions will be added.
 If a topic already exists with a larger number of partitions than the maximum of (`minPartitionCount` and `partitionCount`), the existing partition count will be used.
-====
\ No newline at end of file
+====
+
+=== Usage examples
+
+In this section we illustrate the use of the above properties for specific scenarios.
+
+==== Example: security configuration
+
+Apache Kafka 0.9 supports secure connections between client and brokers.
+For configuring the binder to take advantage of this feature, follow the guidelines in the http://kafka.apache.org/090/documentation.html#security_configclients[Apache Kafka Documentation], using the `spring.cloud.stream.kafka.binder.configuration` option to set the consumer properties for all clients created by the binder.
```

Comment:
"option to set the consumer properties for all clients created by the binder" -> since this property is a generic setting to both producers and consumers, should we remove "consumer"?